RELEASES.md

```diff
@@ -1,8 +1,22 @@
-# 0.4.0-beta1 (2023-05-10)
+# 0.4.0 LTS (2023-05-28)
 
-Changes compared to v0.3.5:
+No changes compared to v0.4.0-beta2.
 
-## Breaking Changes
+## Announcements
+
+The `v0.4.x` LTS series will be the last release to support older CMake and Rust versions.
+If necessary, fixes will be backported to the v0.4 branch. New features will not be
+actively backported after the v0.5 release, but community contributions are possible.
+The `v0.4.x` series is currently planned to be maintained until the end of 2024.
```

Comment:
I'm glad that you are willing to maintain this. If you think you can do it. 
I would be careful to promise anything though, that you may regret later. That said, A line in the ChangeLog doesn't make anyone liable for anything so it's fine to communicate your intend even if we stop maintaining this branch earlier.  I personally would leave the exact date out. Up to you

Reply:
I would say I was quite careful with what I promised: Fixing bugs in the LTS branch and reviewing pull-requests for feature backports. The end of 2024 is in 1.5 years, so the timeframe I promised is also not overly long.